.github/workflows/release.yml

```diff
@@ -65,3 +68,53 @@ jobs:
           done
         done
         hub release create "${ASSET_FLAGS[@]}" -F ${GITHUB_WORKSPACE}/release-note.txt --draft "${RELEASE_TAG}"
+  
+  containerize:
+    runs-on: ubuntu-20.04
```

Comment:
Can we publish images only on every tag instead of every push for avoiding unstable code published?